.rubocop_todo.yml

```diff
@@ -24,6 +24,11 @@ Metrics/AbcSize:
     - lib/new_relic/cli/commands/deployments.rb
     - test/**/*
 
+# Offense count: 1
+Metrics/CollectionLiteralLength:
+  Exclude:
+    - 'lib/new_relic/agent/configuration/default_source.rb'
```

Comment:
[Reminder](https://github.com/newrelic/newrelic-ruby-agent/pull/2369#discussion_r1428446469) to inline disable this cop before finalizing.

Reply:
Done.